Firestore/Source/Local/FSTLevelDBKey.mm

```diff
@@ -29,6 +29,7 @@
 using Firestore::StringView;
 using leveldb::Slice;
 
+static const char *kVersionTableGlobal = "version";
```

Comment:
The target global table is `kTargetGlobalTable` and this is `kVersionTableGlobal`. Could you make this consistent?

Reply:
Done.